app/controllers/restaurants_controller.rb

```diff
@@ -23,6 +23,13 @@ def create
 		end
 	end
 
+	def destroy
+		@restaurant = Restaurant.find(params[:id])
+		@restaurant.destroy
+
+		redirect_to restaurants_path
+	end
```

Comment:
a it nitpicky, but I would love error handling here if this fails. If this is the only issue on this PR I will merge it